WildFly_29.0.0.Final/jakarta-full-platform-jdk17.adoc

```diff
@@ -0,0 +1,687 @@
+
+TCK result summary:
+----
+[INFO] Running TestSuite
+[INFO] Tests run: 149, Failures: 0, Errors: 0, Skipped: 0
+----
+
+https://download.eclipse.org/jakartaee/cdi/4.0/cdi-tck-4.0.10-dist.zip[Jakarta Contexts and Dependency Injection 4.0.10 TCK]
+
+SHA-256: `9b14911f6324b8f3869fd3faa01f3ef9eeab7bfd0734477555385b9f9120d787`
+
+TCK result summary:
+----
+[INFO] Tests run: 1833, Failures: 0, Errors: 0, Skipped: 0
+----
+
+https://download.eclipse.org/jakartaee/debugging/2.0/jakarta-debugging-tck-2.0.0.zip[Jakarta Debugging Support for Other Languages 2.0 TCK]
+
+SHA-256: `71999815418799837dc6f3d0dc40c3dcc4144cd90c7cdfd06aa69270483d78bc`
+
+TCK result summary:
+----
+Tests run: 1, Failures: 0, Errors: 0
+----
+
+https://download.eclipse.org/jakartaee/dependency-injection/2.0/jakarta.inject-tck-2.0.2-bin.zip[Jakarta Dependency Injection 2.0.2 TCK]
+
+SHA-256: `23bce4317ca061c3de648566cdf65c74b57e1264d6891f366567955d6b834972`
+
+TCK result summary:
+----
+[INFO] Tests run: 50, Failures: 0, Errors: 0, Skipped: 0
+----
+
+https://download.eclipse.org/jakartaee/faces/4.0/jakarta-faces-tck-4.0.2.zip[Jakarta Faces 4.0.2 TCK] 
+
+SHA-256: `82a897b7e55a54be7366495ae2e12794a7a63a4e9531d323b071eedf92f2c66b`
+
+TCK result summary:
+----
+Tests run: 252, Failures: 0, Errors: 0
+----
+----
+[javatest.batch] Completed running 5400 tests.
+[javatest.batch] Number of Tests Passed      = 5400
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+----
+
+https://download.eclipse.org/jakartaee/jsonb/3.0/jakarta-jsonb-tck-3.0.0.zip[Jakarta JSON Binding 3.0 TCK]
+
+SHA-256: `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b`
+
+Includes already certified [Eclipse Yasson 3.0.2](https://github.com/eclipse-ee4j/yasson/releases/tag/3.0.2)
+
+https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.0.zip[Jakarta JSON Processing 2.1.0 TCK] 
+
+SHA-256: `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
+
+Includes already certified [Eclipse Parsson 1.1.3](https://github.com/eclipse-ee4j/parsson/releases/tag/1.1.3)
+
+https://download.eclipse.org/jakartaee/mail/2.1/jakarta-mail-tck-2.1.0.zip[Jakarta Mail 2.1.0 TCK] 
+
+SHA-256: `6f02a92e0a5ef60260e65f95938cc566da2f93a3d269c3b321da0d787a3448a5`
+
+Includes already certified [Eclipse Angus Mail 2.0.2](https://github.com/eclipse-ee4j/angus-mail/releases/tag/2.0.2)
+
+https://download.eclipse.org/jakartaee/restful-ws/3.1/jakarta-restful-ws-tck-3.1.3.zip[Jakarta RESTful Web Services 3.1.3 TCK]
+
+SHA-256: `8ff1e6ee4165ca5444cb181a4a70fcde94b9b3915cad948c7f1a029290b09773`
+
+TCK result summary:
+----
+Tests run: 2796, Failures: 0, Errors: 0, Skipped: 128
+----
+
+https://download.eclipse.org/jakartaee/security/3.0/jakarta-security-tck-3.0.0.zip[Jakarta Security 3.0.0 TCK] 
+
+SHA-256: `696776046dfeaed74266a5d1c4dac7fea5437b6f51743b7fe10962dde755ff8f`
+
+TCK result summary:
+----
```

Comment:
We're missing the section for the 115 new TCK tests; below is the old tck section.

Reply:
Good catch!